src/components/MapFacetContents.js

```diff
@@ -31,11 +31,20 @@ type MapFacetContentsProps = {
   size: any;
   /** The public key with which to connect to the mapbox public apis. */
   mapboxKey: string;
+  /** Custom location pointer glyphicon, default is the map-marker glyphicon */
+  glyph: string | null;
+  /** Custom location pointer image uri, if available image is shown instead of glyphicon */
+  uri: string | null;
```

Comment:
Can you rename this to "pointerImageUri"? And rename glyph to "pointerGlyph"?

Reply:
Done